modules/virt-hot-plugging-cpu.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-edit-vms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-hot-plugging-cpu_{context}"]
+
+= Hot plugging CPUs on a virtual machine
+
+You can add or remove the number of CPU sockets allocated to a virtual machine (VM) without having to restart the VM by using the {product-title} web console.
```

Comment:
@jean-edouard the word "sockets" is completely correct, but is it needed in this context?

Reply:
I'm not sure I understand the question, but yes I think it's important to tell the user we hotplug sockets and not CPUs, because that means hotplugging 1 (socket) will most likely result in more than 1 CPU getting hotplugged.